src/components/wallet-box.tsx

```diff
@@ -0,0 +1,25 @@
+import { useWalletContext } from '../contexts/wallet.context';
+import { useSession } from '../hooks/session.hook';
+import StyledDataBox from '../stories/StyledDataBox';
+import StyledDataTextRow from '../stories/StyledDataTextRow';
+
+export function WalletBox(): JSX.Element {
+  const { isConnected } = useWalletContext();
+  const { address, blockchain } = useSession();
+
+  function blankedAddress(): string {
+    return `${address?.slice(0, 6)}...${address?.slice(address?.length - 5)}`;
+  }
```

Comment:
Can we use CSS text-overflow ellipsis instead?

Reply:
need to look up how ;)

Reply:
imo overflow ellipsis only shortens at the end. 

since crypto adresses usually get checked buy comparing the first and the last digits, it would be optimal to have a format like 

0x35423...f345d

Reply:
would be possible, but we would need to define a size for the right label & write a custom css class as tailwind doesn't support `text-overflow-middle`

Reply:
Let's keep it like this